examples/examples-by-storage/main.py

```diff
@@ -26,81 +32,94 @@ def main(modelstore_in):
     # directory
     tmp_dir = tempfile.mkdtemp()
 
-    # In this demo, we train two models, so that we can demonstrate
+    # In this demo, we train a models, so that we can demonstrate
     # how modelstore keeps track of uploaded models for us
```

Comment:
I've simplified the demo below, to only train a single model and upload it, change its state several times, and eventually delete it.